File: examples/web-mvc-i8n/web-dotnet/Web/Common/Utils/WebUtils.cs
namespace Web.Common.Utils;

public static class WebUtils
{
    public const string MsgSuccess = "MSG_SUCCESS";
    public const string MsgInfo    = "MSG_INFO";
    public const string MsgError   = "MSG_ERROR";

    public static string GetMessage(string key, params object[] args)
        => Web.Resources.Lang.MessageHelper.GetMessage(key, args);
}

File: web-spring/src/main/java/io/github/agussyahrilmubarok/web/common/WebUtils.java
package io.github.agussyahrilmubarok.web.common;

import jakarta.servlet.http.HttpServletRequest;
import org.springframework.stereotype.Component;
import org.springframework.web.context.request.RequestContextHolder;
import org.springframework.web.context.request.ServletRequestAttributes;

@Component
public class WebUtils {

    public static final String MSG_SUCCESS = "MSG_SUCCESS";
    public static final String MSG_INFO = "MSG_INFO";
    public static final String MSG_ERROR = "MSG_ERROR";

    public static HttpServletRequest getRequest() {
        return ((ServletRequestAttributes) RequestContextHolder.getRequestAttributes()).getRequest();
    }
}
